app/views/posts/show.html.erb

```diff
@@ -1,2 +1,8 @@
-<h1>Single Posts</h1>
-<p>This is to display all single posts</p>
+<section class = "post-section-container">
+    <p> <%= @post.title %> </p>
+    <p> Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+    <p> <%= @post.text %> </p>
+    <% @post.comments.each do |comment| %>
+    <p> <%= comment.user.name %> : <%= comment.text %> </p>
+    <% end %>
+</section>
```

Comment:
- [ ] Let's 'also ensure that `Post show` page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)